chemprop/args.py

```diff
@@ -339,8 +467,10 @@ class TrainArgs(CommonArgs):
     """Whether to add bias to linear layers."""
     hidden_size: int = 300
     """Dimensionality of hidden layers in MPN."""
+    mpn_encoder_middle_dim: int = 300
+    """Dimensionality of middle hidden layers in large MPN."""
     depth: int = 3
-    """Number of message passing steps."""
+    """Number of message passing steps + 1."""
```

Comment:
This is changing the default behavior of chemprop? This would be a significant change, if so.

(cc @kevingreenman)

Reply:
I don't think this is an attempt to change the default behavior, but may be a response to confusion over how this argument is used in the code [here](https://github.com/chemprop/chemprop/blame/b08063bc412f898ea2dbaf478b52ae9914604f84/chemprop/models/mpn.py#L120). Since we do message passing `for depth in range(self.depth - 1)`, I think this correction to the CLI description might be appropriate.

Reply:
Since range starts at 0 though, if I pass depth=3 I still get depth=3 (right?). I think changing the description to this contradicts that.

Reply:
If you pass `depth = 3`, it will do `for depth in range(2)`, which will be `depth=0` and `depth=1`, i.e. two message-passing iterations, right?

Reply:
Ah duh, thanks for typing that out for me 😅 

This seems... bad? Like we should change the `for` loop not the CLI description? Looks like we made a fencepost error in that loop when it was first written.

Reply:
I had to think through it several times myself 😅 I agree, changing the `for` loop might make more sense, but I'm hesitant to change something significant like that right before sunsetting v1, so maybe updating the description is as far as we should go.